src/lib/y2storage/devicegraph.rb

```diff
@@ -166,6 +166,12 @@ def filesystems
       Filesystems::Base.all(self)
     end
 
+    # @return [Boolean]
+    def filesystem_in_network?(mountpoint)
+      filesystem = filesystems.find { |i| i.mountpoint == mountpoint }
```

Comment:
What if there is no such mountpoint? `filesystem` would be nil and this would explode.... which makes me think we miss good unit tests here.